LogMonitor/src/LogMonitor/LogFileMonitor.cpp

```diff
@@ -810,7 +811,7 @@ LogFileMonitor::LogFilesChangeHandler()
     const DWORD eventsCount = 3;
 
     LARGE_INTEGER liDueTime;
-    INT64 millisecondsToWait = 30000LL;
+    INT64 millisecondsToWait = 1000LL;
     liDueTime.QuadPart = -millisecondsToWait*10000LL; // wait time in 100 nanoseconds
 
     HANDLE timerEvent = CreateWaitableTimer(NULL, FALSE, NULL);

```

Comment:
PS: Looking at this file, I noticed that this timer is not closed or canceled. This could lead to resource leaks. Is it possible to address this in this PR?

Reply:
@TinaMor  the timer is cancelled here:
![image](https://github.com/microsoft/windows-container-tools/assets/28774968/54c52be8-b67b-4a34-bb51-39ceb967dc37)

I will be creating a separate issue & PR to handle file monitor optimization.

Reply:
Thank you for pointing this out. Timer has been closed after the while loop.